src/pages/LoadingScreen.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div>
```

Comment:
Is the wrapping `<div>` actually necessary? Seems to be quite irrelevant... (trying to keep the amount of elements in the browser low)

Reply:
This one is necessary. The loader rotates and we don't want the entire page to rotate — because so would every element you add to it. Otherwise [this](https://eutopiaworkspace.slack.com/files/UFPRQC11S/FJ9NB5VBL/screen_recording_2019-05-12_at_16.17.03.mov) happens.